Project.toml

```diff
@@ -12,7 +12,10 @@ ClimateTools = "4f4ee721-4970-5af2-8560-6c1d960e3231"
 Conda = "8f4d0f93-b110-5947-807f-2305c1781a2d"
 CondaBinDeps = "a9693cdc-2bc8-5703-a9cd-1da358117377"
 Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
+GeoMakie = "db073c08-6b98-4ee5-b6a4-5efafb3259c6"
 Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
+Makie = "ee78f7c6-11fb-53f2-987a-cfe4a2b5a57a"
```

Comment:
You may want to change these to CairoMakie and AbstractPlotting, which don't bring in the GLMakie dependency.

Reply:
Shouldn't we only depend on AbstractPlotting and GeoMakie, without enforcing a backend? That's what I do in InteractiveChaos for example (no GeoMakie there)

Reply:
Yep, that's also doable, there aren't any explicit ties to CairoMakie.